src/containerapp/azext_containerapp/_utils.py

```diff
@@ -1506,24 +1506,7 @@ def is_registry_msi_system(identity):
 
 
 def validate_environment_location(cmd, location):
-    from ._constants import MAX_ENV_PER_LOCATION
```

Comment:
should we also remove MAX_ENV_PER_LOCATION in constants file?

Reply:
Removed. Also remove the last ValidationError which reference this variable. Can you please have a look again?